docker-compose-citus.yml

```diff
@@ -9,11 +9,37 @@
 # $ docker-compose -f docker-compose-citus.yml up -d
 # You can read more about it in the:
 # https://github.com/patroni/patroni/blob/master/docker/README.md#citus-cluster
-version: "2"
-
 networks:
     demo:
 
+x-pg-healthcheck: &pg-healthcheck
+  test: ["CMD-SHELL", "pg_isready -U postgres"]
+  interval: 10s
+  retries: 5
+  start_period: 10s
+
+x-etcd-healthcheck: &etcd-healthcheck
+  test: ["CMD-SHELL", "etcdctl endpoint health"]
+  interval: 10s
+  retries: 5
+  start_period: 10s
+
+x-pg-resource-limit: &resource-limit-psql
+  limits:
+    cpus: "0.5"
+    memory: 512M
+  reservations:
+    cpus: "0.1"
+    memory: 128M
+
+x-etcd-resource-limit: &resource-limit-etcd
+  limits:
+    cpus: "0.5"
+    memory: 256M
+  reservations:
+    cpus: "0.1"
+    memory: 64M
```

Comment:
These resource limits are way to low.
Better not to have anything at all.

Again, this compose file exists just to play with Citus managed by Patroni, not for production.
If you want to have something production-ready - you can do it in your own repo.